storage/container_storage_interface/persistent-storage-csi-aws-efs.adoc

```diff
@@ -0,0 +1,55 @@
+[id="persistent-storage-csi-aws-efs"]
+= AWS Elastic File Service CSI Driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-aws-efs
+
+toc::[]
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for AWS Elastic File Service (EFS).
+
+:FeatureName: AWS EFS Driver Operator
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a CSI Operator and driver.
+
+After installing the AWS EFS CSI Driver Operator, {product-title} installs the AWS EFS CSI Operator and the AWS EFS CSI driver by default in the `openshift-cluster-csi-drivers` namespace. This allows the AWS EFS CSI Driver Operator to create CSI-provisioned PVs that mount to AWS EFS assets.
+
+* The _AWS EFS CSI Driver Operator_, after being installed, does not create a storage class by default to use to create persistent volume claims (PVCs). However, you can manually create the AWS EFS `StorageClass`.
+The AWS EFS CSI Driver Operator supports dynamic volume provisioning by allowing storage volumes to be created on-demand, eliminating the need for cluster administrators to pre-provision storage.
+
+* The _AWS EFS CSI driver_ enables you to create and mount AWS EFS PVs. The AWS EFS CSI driver must be manually installed.
+
+include::modules/persistent-storage-csi-about.adoc[leveloffset=+1]
+
+== Installing the AWS EFS Driver Operator
+
+The AWS EFS Driver Operator is not installed by default. You must install both the Operator and driver.
+
+:FeatureName: AWS EFS
+include::modules/persistent-storage-csi-olm-operator-install.adoc[leveloffset=+2]
```

Comment:
This shows "Installing the AWS EFS Driver Operator" and "Installing the AWS EFS CSI Operator" chapters just next to each other. IMO one of them should be removed (like, remove the chapter header, not the content).